pom.xml

```diff
@@ -368,12 +369,24 @@ SOFTWARE.
           <plugin>
             <groupId>com.qulice</groupId>
             <artifactId>qulice-maven-plugin</artifactId>
-            <version>0.17.1</version>
+            <!--
+              @todo #837:30min Upgrade qulice to latest 0.17.x and if possible
```

Comment:
@victornoel please also mention to review all diamond usage, there was a fix in qulice 0.17.5, that should not require explicit types on the right hand said of diamond, e.g. in BkTimetable line 65:
```
this.threads = new ConcurrentHashMap<Thread, Long>(1);
```

Should be:
```
this.threads = new ConcurrentHashMap<g>(1);
```

There are about 70 cases like this.

Reply:
@krzyk do you think this should be a second `@todo`?